scripts/update-codegen.sh

```diff
@@ -18,17 +18,27 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+SCRIPT_ROOT=$(cd `dirname "${BASH_SOURCE[0]}"`/.. && pwd)
+[[ $# -eq 1 ]] && VERIFY_ONLY=true || VERIFY_ONLY=false
+TMP_DEST="${1:-${SCRIPT_ROOT}/_tmp_dest}"
```

Comment:
What about having an optional argument indicating the output base, and using SCRIPT_ROOT or current directory if none is provided? That way you would not need to copy but could still, if wanted, generate into a temp location.

Reply:
If the `--output-base` is set, then code will be generated under:
`<output-base-dir>/github.com/skupperproject/skupper/pkg/{apis,generated}`.

Therefore using `${SCRIPT_ROOT}` would cause it to create this entire tree under the project's root dir.

If we do not provide the `--output-base` at all, then it will perform the updates straight into:
`${GOPATH}/src/github.com/skupperproject/skupper/pkg/`, which would work for people working
under `$GOPATH/src`, but not for people working outside of it.

Reply:
What if we also use `--trim-path-prefix github.com/skupperproject/skupper/`?

Reply:
Let me give it a try.

Reply:
Keeping as is because `--trim-path-prefix` is not available with v0.17.0.